cssselect2/parser.py

```diff
@@ -266,14 +267,18 @@ def skip_whitespace(self):
             if peek is None or peek.type != 'whitespace':
                 break
             self.next()
+            if u'\n' in peek.value:
+                self.lines += 1
+
             has_whitespace = True
-        return has_whitespace
+        return has_whitespace, self.lines
 
 
 class Selector(object):
-    def __init__(self, tree, pseudo_element=None, extensions=None):
+    def __init__(self, tree, pseudo_element=None, extensions=None, src_line=0):
         self.parsed_tree = tree
         self.extensions = extensions
+        self.source_line = src_line
```

Comment:
It appears (from https://github.com/Connexions/cnx-easybake/pull/45 ) that this field is actually an offset (but maybe I am reading the code wrong); is that correct?